### DISCUSSION TITLE
My application keeps getting rejected even though I meet all the requirements

### DISCUSSION THREAD

**Discussion by raku2368** [community/community#61104] [GitHub Education]
I am trying to apply for an edu program, but my application has been rejected 4 times because the image does not contain the name of the school.  I am trying to authenticate by obtaining a legally valid official certificate of enrollment from the Korean government's website, and the name of my middle school is written very large on the document.  May I know why my application is being rejected?

**Comment by samus-aran** [comment]
Hi @raku2368 

You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Ensure you have selected the correct school in your application. If your school is not available as a selection you can proceed with entering the schools details only if you feel the schools identified on the map do not match yours.

Let us know how you get on. 👍

**Reply by raku2368** [reply]
![image](https://github.com/community/community/assets/85862456/85131f99-1b30-45c6-b3c6-c9e2bad3deb6)
I made another 6th request with my official transcript in English, but it was also rejected for the same reason.
What the hell do I have to fix to get my request approved?
![image](https://github.com/community/community/assets/85862456/cdc11596-e7f3-48de-8792-b7cfa46a0ac9)
